content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -1177,6 +1183,33 @@ The `fieldPath` field does not support indexing arrays numerically.
 
 Setting `fieldPath` is optional.
 
+#### The `optionalOldSelf` field {#field-optional-oldself}
+
+The `optionalOldSelf` field is a boolean field added in Kubernetes 1.29. The feature
+[CRDValidationRatcheting](#validation-ratcheting) must be enabled in order to 
```

Comment:
We may want to link the feature name to the `/docs/reference/command-line-tools-reference/feature-gates/`.

Reply:
The `#validation-ratcheting` tag already contains the direct link to feature gate reference:

https://github.com/kubernetes/website/blob/d820f2b988d90756c11b0c9371b4a830e7cc6c66/content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md?plain=1#L724-L727

IMO id like to keep this link as is to link the general "CRDValidationRatcheting" documentation on the same page to the `optionalOldSelf` field which is part of the feature, but listed with the rest of the CEL CRD fields.